perses/annihilation/relative.py

```diff
@@ -800,8 +800,10 @@ def _add_nonbonded_force_terms(self, add_custom_sterics_force=True):
         sterics_custom_nonbonded_force = openmm.CustomNonbondedForce(total_sterics_energy)
         if self._softcore_LJ_v2:
             sterics_custom_nonbonded_force.addGlobalParameter("softcore_alpha", self._softcore_LJ_v2_alpha)
+            sterics_custom_nonbonded_force.setCutoffDistance(self.r_cutoff)
```

Comment:
I just realized that this should probably exist right after line 800 where the object is instantiated in a single statement, instead of being inside the conditional.

Reply:
Also, I don't think there's a instance attribute for `r_cutoff`, we probably want to use the one defined in lines 767 or 775, that in turn don't need to exist separately, these could be computed outside that conditional.

Reply:
Oh yeah, that's why the CI tests aren't passing https://github.com/choderalab/perses/actions/runs/5955395011/job/16153949168?pr=1230

Reply:
yes much better!